### ISSUE TITLE
Feature Request: Support for loading Georam files

### ISSUE THREAD

**Issue by Chris5523** [MiSTer-devel/C64_MiSTer#133]
This game https://psytronik.itch.io/lykia supports Georam, it is possible to load all 5 discs into the Georam and a readymade 1MB Georam file is provided with the game. It would be nice to have the option to load Georam files besides the support for loading REU files. It is a rather niche request, I don't think there are many games supporting the Georam in this way.

**Comment by sorgelig** [comment]
Hot it works on real HW?

**Comment by Chris5523** [comment]
On real hardware there is a seperate disc set to store the discs in the Georam. As there is battery backed Georam it is possible to store the game data this way and to access the game directly with an also supplied prg.

I couldn't try it myself as I have no C64 with Georam, but it seems a cool feature because the Georam is so much faster than the normal C64 RAM.

**Comment by sorgelig** [comment]
ironically this game is crippled when georam is used. It doesn't allow to save to disk, so whole georam should be saved. I don't know why developer choose such way as it could just use disk to save..

**Comment by Chris5523** [comment]
Yeah, this is a really strange decision. I guess as many Georam expansions are battery powered the developer tries to use it as some kind of cartridge.
